README.rst

```diff
@@ -38,6 +38,13 @@ Docs site`_.
 
 .. _our Read the Docs site: https://google-cartographer.readthedocs.io
 
+Community
+=========
+
+Please join the `mailing list`_ and ask questions.
```

Comment:
This is the same call to action that Ceres uses. I prefer it since it encourages folks to join even if they don't currently have any questions.